### PR TITLE
v0.1 update citi bike doc and data version

### PR DESCRIPTION
**Pull request by zhawan**
# Description

Kuanwei found data preparation part of citi_bike quick start doc was outdated.
Updated the doc. 
Jinyu found data files would not auto deployed if user has a older MARO version.
Updated the data version.

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [ ] New feature
- [ ] Test
- [v] Doc update
- [ ] Docker update

## Related Component

- [v] Simulation toolkit
- [ ] RL toolkit
- [ ] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [ ] Linux
- Python version:
  - [ ] 3.6
  - [ ] 3.7
- Key information snapshot(s):
  
## Needs Follow Up Actions

- [ ] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [v] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: maro/__misc__.py [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 __version__ = "0.1.1a11"
 
-__data_version__ = "0.1"
+__data_version__ = "0.2"
```
Comment (Jinyu-W) [inline maro/__misc__.py:7]: Is the data version independent to the project version?

File: docs/source/scenarios/citi_bike.rst [hunk 2]
```diff
@@ -563,19 +563,19 @@ For the example above, the directory structure should be like:
    |-- ~/.maro
```
Comment (Jinyu-W) [inline docs/source/scenarios/citi_bike.rst:563]: Could you add a part to introduce the two kinds of usage way of citi bike at the begining of "Data Preparation"? I.e:
- Manually call: maro env data generate ...  The advantage is...
- Data can be automatically downloaded and generated if directly call Env(scenario="citi_bike", topology="...")

**Review by Jinyu-W — CHANGES_REQUESTED** [review]

File: maro/__misc__.py (hunk 1 above)
Comment (zhawan) [reply maro/__misc__.py:7]: yes

**Comment by ArthurJiang** [comment]
Add a meaningful title for this pr.

File: docs/source/scenarios/citi_bike.rst [hunk 3]
```diff
@@ -516,7 +516,10 @@ Data Preparation
 ^^^^^^^^^^^^^^^^
 
 To start the simulation of Citi Bike scenario, users need to first generate the
-related data. Below is the introduction to the related commands:
+related data. (If the simulator did not detect the related data, it will automatically
+try to generate the related data in a temp folder, and remove the temp data after the
+experiment. So we encourage the users to generate the related data manually first.)
+Below is the introduction to the related commands:
```
Comment (Jinyu-W) [inline docs/source/scenarios/citi_bike.rst:523]: To start the simulation of Citi Bike scenario, users have two options for the data preparation:

1. If the topology data is not generated in advance, the system will automatically download and process the relevant data when the environment is created. The data will be stored in a temporary folder and be automatically deleted after the experiment.

2. Before creating the environment, users can also manually download and generate relevant data. This approach will save you a lot of time if you need to conduct several experiments on the same topology. Therefore, we encourage you to generate the relevant data manually first.

The following is an introduction to related commands:

Comment (Jinyu-W) [reply docs/source/scenarios/citi_bike.rst:523]: How about changing to this?

**Review by Jinyu-W — APPROVED** [review]

**Pull request merged**